index.html

```diff
@@ -5,16 +5,79 @@
   <title>Quantified Self - Diary</title>
 </head>
 <body>
-  <article class="food-headers">
-    <p class="name-header">
-      <strong>Name</strong>
-    </p>
-    <p class="calorie-header">
-      <strong>Calories</strong>
-    </p>
-    <div class="hidden"></div>
-  </article>
-  <div id="food-table-info"></div>
+  <h2>Main Diary</h2>
+  <div class="meals-container">
+    <div class="meals-diary-column-1">
```

Comment:
I like that we're utilizing flex here.

One option to consider in the future is to use the CSS grid properties. I haven't worked with them much, so the learning curve would be steeper, but it's an option we have in our toolbelt!